service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -16,9 +16,21 @@
 
 import java.util.Arrays;
 
+/**
+ * Lagom service implementations must create one implementation of this interface and use it to bind Service
+ * implementations.
+ */
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
 
-    // redefines bindServiceInfo from super interface.
+    /**
+     * Creates a custom {@link ServiceInfo} for this Lagom service. This method overrides
+     * {@link ServiceClientGuiceSupport#bindServiceInfo(ServiceInfo)} with custom behavior for consume-only Lagom
+     * services.
+     * <p>
+     * The customization may be used by Lagom to add features only applicatble to Lagom services transparently.
```

Comment:
I'm in favor of keeping our options open, but still think that an example could help clarify what the statement means.

Anyway, it might be a moot point if we intend to deprecate multiple service bindings anyway. It's up to you.